src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -71,19 +71,18 @@ public Office365ConnectorWebhookNotifier(Run run, TaskListener listener) {
     }
 
     public void sendBuildStartedNotification(boolean isFromPreBuild) {
+        WebhookJobProperty property = (WebhookJobProperty) job.getProperty(WebhookJobProperty.class);
+        if (property == null || property.getWebhooks() == null || property.getWebhooks().isEmpty()) {
```

Comment:
this code is duplicated so nice to separate this validation into nicer place such as method